build.gradle.kts

```diff
@@ -38,13 +38,13 @@ repositories {
 }
 
 allprojects {
-    if (this.name != "exposed-tests" && this.name != "exposed-bom" && this != rootProject) {
+    if (this.name != "exposed-tests" && this.name != "exposed-bom" && this.name != "exposed-version-catalog" && this != rootProject) {
```

Comment:
@jetbrains-junie `exposed-version-catalog` needs to be published so do not filter it here.